### ISSUE TITLE
only <code> is not working

### ISSUE THREAD

**Issue by Willem-Siebe** [jes-carr/wordpress-prettify-code-syntax#1]
Hi, 

When I use (I replaced the < sign here because the post edits it)  [code class="prettify language-html"]Code here[/code] the plugin is not working for me. Do you experience the same?

Kind regards,

Willem

**Comment by jes-carr** [comment]
Sorry I didn't get it. What is the code that you are using? Can you be a bit more specific?

**Comment by Willem-Siebe** [comment]
Hi, actually everything is in my first post, but let me structure it a little bit more.

First of all, I replaced the '<' for '[' and the '>' for ']' in this post.

When I use your plugin I encounter problems when I use this in my posts/pages:

[code class="prettify language-html"][p]This is a paragraph.[/p][/code]

Please let me know if I have to explain myself better.

**Comment by jes-carr** [comment]
Gotcha now. That should work, but I'll check and let you know if I can reproduce it. Are you using the HTML editor?

(by the way, in the comments you can add code using "`" around the code, or preceding with 4 spaces to have a code block. Check the GitHub Flavored Markdown link at the top right)

**Comment by Willem-Siebe** [comment]
Yes, always use the html editor! Thanks for the tip for github code.

**Comment by Raboo** [comment]
I had the same problem, in the editor it was like this:
`<pre class="prettyprint"><code class="language-php">// my code</code></pre>`
Output was like this(but with pretty colors):
`<code class="language-php">// my code`
